### ISSUE TITLE
Cannot able to access "+ and -" buttons in Font size of general tab of preferences using keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#377]
### Steps to Reproduce
1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Navigate to Tools -> Preferences and invoke it.
3. It opens the Genral tab of preferences and navigate to Font size text editor. Try to set the focus on "+ and -" buttons and invoke them using keyboard.

### Expected Behavior
Should be able to access the "+ and -" buttons using keyboard as they are accessible by mouse.
### Actual Behavior
Cannot able to set focus on "+ and -" buttons in Font size of general tab of preferences and cannot able to invoke them using Keyboard.
Note: Font size can be increased/decreased using UP/Down arrows.

Narrator Behavior: Narrator sets focus on "+/-" buttons and reads as button.
NVDA Behavior: Cannot able to set focus on "+/-" buttons using nvda.
### User Impact:
User cannot able to invoke "+ and -" buttons which are accessible by mouse.
### Screenshot:
![and -](https://user-images.githubusercontent.com/22956130/40760926-b201edae-64b6-11e8-92aa-8a3d3ac2b3dc.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
